src/components/FlowRunTimelineSelectionPanel.vue

```diff
@@ -51,16 +53,42 @@
 
   const api = useWorkspaceApi()
   const routes = useWorkspaceRoutes()
-  const taskRunIdArgs = computed<[string] | null>(() => taskRunId.value ? [taskRunId.value] : null)
-  const taskRunSubscription = useSubscriptionWithDependencies(api.taskRuns.getTaskRun, taskRunIdArgs)
-  const taskRun = computed(() => taskRunSubscription.response)
+  const subscriptionArgs = computed<[string] | null>(() => id.value ? [id.value] : null)
```

Comment:
You shouldn't need this check or `useSubscriptionWithDependencies` where it's used later - your `id` is a require property on the component. If other places dictate that it actually isn't required, I don't think I understand what this component is meant to render, currently it won't render anything without an id.

Reply:
So it appears the `id`, and the entire content really, is optional for the sake of animating the panel away. When a selection is cleared, the panel is animated out of view, during that time, the selection is null but the panel still needs to render. Thinking through other ways of handling this..